src/main.cpp

```diff
@@ -138,78 +126,48 @@ void DDWAF::dispose(const Napi::CallbackInfo& info) {
   return this->Finalize(info.Env());
 }
 
-void DDWAF::updateRuleData(const Napi::CallbackInfo& info) {
-  mlog("Updating rule data on DDWAF");
+void DDWAF::update(const Napi::CallbackInfo& info) {
   Napi::Env env = info.Env();
-  if (this->_disposed) {
-    Napi::Error::New(env, "Could not update rule data on a disposed WAF").ThrowAsJavaScriptException();
+
+  size_t arg_len = info.Length();
```

Comment:
i usually only put info.length() in it's own variable if it's used more than once, usually it's just inlined like `if (info.Length() < 1)`